update.go

```diff
@@ -108,6 +108,10 @@ other options are ignored.
 			Name:  "memory-swap",
 			Usage: "Total memory usage (memory + swap); set '-1' to enable unlimited swap",
 		},
+		cli.StringFlag{
```

Comment:
I want `int64` though, it doesn't exist unfortunately hence why the others are using `StringFlag`